src/app/(screens)/room-report.tsx

```diff
@@ -0,0 +1,262 @@
+                                        style={styles.inputLabel}
+                                    >
+                                        {t(
+                                            `pages.rooms.report.category.type.${categroy}`
+                                        )}
+                                    </DropdownMenuItemTitle>
+                                </DropdownMenuItem>
+                            )
+                        })}
+                    </DropdownMenuContent>
+                </DropdownMenu.Root>
+
+                <Text style={styles.inputLabel}>
+                    {t('pages.rooms.report.description.title')}
+                </Text>
+                <TextInput
+                    editable
+                    multiline
+                    numberOfLines={4}
+                    maxLength={2000}
+                    style={{
+                        ...styles.textInput,
+                        ...styles.multilineTextInput,
+                    }}
+                    placeholder={t(
+                        'pages.rooms.report.description.placeholder'
+                    )}
+                    value={description}
+                    onChangeText={(text) => setDescription(text)}
+                />
+
+                <TouchableOpacity
+                    disabled={submitDisabled}
+                    onPress={() => {
+                        if (!reportCategory || !description || !room) return
+                        mutation.mutate({
+                            reason: reportCategory,
+                            description,
+                            room: room,
+                        })
+                    }}
+                    style={styles.submitButton(submitDisabled)}
+                >
+                    {loading ? (
+                        <ActivityIndicator
+                            color={getContrastColor(theme.colors.primary)}
+                            size={15}
+                        />
+                    ) : (
+                        <Text style={styles.buttonText(submitDisabled)}>
+                            {t('submit')}
+                        </Text>
+                    )}
+                </TouchableOpacity>
+            </View>
+        </ScrollView>
+    )
+}
+
+const stylesheet = createStyleSheet((theme) => ({
+    buttonText: (disabled: boolean) => ({
+        fontWeight: 'bold',
+        fontSize: 15,
+        color: disabled
+            ? UnistylesRuntime.themeName === 'dark'
+                ? Color(getContrastColor(theme.colors.primary))
+                      .lighten(0.1)
+                      .hex()
+                : Color(getContrastColor(theme.colors.primary))
+                      .darken(0.1)
+                      .hex()
+            : getContrastColor(theme.colors.primary),
+    }),
+    container: {
+        backgroundColor: theme.colors.background,
+        padding: theme.margins.card,
```

Comment:
Use the standard horizontal padding of 12px.

```suggestion
        padding: theme.margins.page,
```